src/commands/trigger-integration-tests-workflow.yml

```diff
@@ -33,8 +33,17 @@ steps:
   - run:
       name: Trigger integration test workflow
       command: |
-        VCS_TYPE=$(echo ${CIRCLE_BUILD_URL} | cut -d '/' -f 4)
-        curl -u ${<<parameters.token-variable>>}: -X POST --header "Content-Type: application/json" -d "{
-          \"branch\": \"${CIRCLE_BRANCH}\",
-          \"parameters\": <<parameters.pipeline-param-map>>
-        }" "https://circleci.com/api/v2/project/${VCS_TYPE}/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/pipeline"
+        VCS_TYPE=$(echo "${CIRCLE_BUILD_URL}" | cut -d '/' -f 4)
+        curl -u "${CIRCLE_TOKEN}": -X POST --header "Content-Type: application/json" -d "{
+            \"branch\": \"${CIRCLE_BRANCH}\",
+            \"parameters\": <<parameters.pipeline-param-map>>
+        }" "https://circleci.com/api/v2/project/${VCS_TYPE}/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/pipeline" -o /tmp/curl-result.txt
+        CURL_RESULT=$(cat /tmp/curl-result.txt)
+
+        if [[ $(echo "$CURL_RESULT" | jq -r .message) == "Not Found" || $(echo "$CURL_RESULT" | jq -r .message) == "Permission denied" || $(echo "$CURL_RESULT" | jq -r .message) == "Project not found" ]]; then
```

Comment:
Should we include `jq/install` in case it's not available?

Reply:
:100: Good call, completely skipped my mind.